server/database/migrations/2022_07_12_072739_create_lessons_table.php

```diff
@@ -17,7 +17,7 @@ public function up()
             $table->id();
             $table->foreignId('word_id');
             $table->foreignId('category_id')->constrained()->cascadeOnDelete();
-            $table->foreignId('result_id');
+            $table->foreignId('result_id')->nullable();
```

Comment:
Bruce, why is `result_id` nullable? then there is already relationship between lessons and results table as stated below so I think no need for this fk na 🤔 
<img width="1318" alt="image" src="https://user-images.githubusercontent.com/70557829/180734102-72861266-e0a4-4510-b0ae-44c148f80c1d.png">

Reply:
ako na gi remove thanks.